packages/zarathustra/skills/asylum_abci_app/behaviours.py

```diff
@@ -449,7 +464,7 @@ def act_from_persona(self):
     def act_from_config(self):
         """Do the act."""
         self.context.logger.info(f"In state: {self._state}")
-        user_data = Path(self.strategy.data_dir) / SPONSOR_CONFIG_FILE
+        user_data = Path(self.strategy.data_dir)
```

Comment:
I think this should work, since `SPONSOR_CONFIG_FILE` not here anymore and we just need `/data` dir